File: _REPO/APRESS/distributed-.net-programming-in-vb-.net/Chapter05/RemotingInterfaces/CustomerServer/CustMain.vb
Imports System.Runtime.Remoting

Module CustomerServer

   Sub Main()
      Console.WriteLine("Customer Server initializing ...")
      RemotingConfiguration.Configure("CustomerServer.exe.config")

      Console.WriteLine("Waiting for clients. Press 'q' to quit")

      Dim input As String
      Do
         input = Console.ReadLine()
      Loop While input <> "q"
   End Sub   
End Module

File: _REPO/APRESS/distributed-.net-programming-in-csharp/Chapter05/RemotingMetadata/CustomerServer/CustMain.cs
using System;
using System.Runtime.Remoting;

namespace CustomerServer
{
   class CustomerMain
   {
      static void Main(string[] args)
      {
         Console.WriteLine("Customer Server initializing ...");
         RemotingConfiguration.Configure("CustomerServer.exe.config");

         Console.WriteLine("Waiting for clients. Press 'q' to quit");

         string input;
         do
         {
            input = Console.ReadLine();
         }while(input != "q");
      }
   }
}
